### ISSUE TITLE
Building intercept should automatically trigger a PBO (re)build

### ISSUE THREAD

**Issue by overfl0** [intercept/intercept#167]
A post-build script should be called that will (re)build the PBO, after building the dll successfully, so that the Intercept library is in a working state as soon as the build finishes.

Optionally, if the PBO doesn't need to be rebuilt because the files have not changed, then maybe have a script that will check for that and only run PBO building when needed?

If the building program (makepbo?) is not present, fail the compilation as Intercept without a PBO is unusable anyway.

**Comment by dedmen** [comment]
>as Intercept without a PBO is unusable anyway
But normally you can use the pbo from the official release.. that said... You won't be able to when we sign our plugins.
Which doesn't apply to Intercept core though.

**Comment by overfl0** [comment]
> But normally you can use the pbo from the official release

Yes, but if you plan on using the official release, you don't need to recompile anything either way :)
Also, a day will come when you will update the sqf scripts and reusing those from the release will not be possible anymore.

True, it's not **really** needed and people can live without that facilitation, but on the other hand... why not make it simpler if we can?
That's one less thing to remember when you're cloning a fresh repo - my main motivation for this ticket. Especially that I tend to delete repos and clone them from scratch quite often.

**Comment by dedmen** [comment]
I don't want builds to fail. Because of something you don't even need. If it's only a warning if makePbo is not there that's good. But I don't want a fail.
